### PR TITLE
Feature/histograms

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

File: src/containers/toputilizers/TopUtilizersReducer.js [hunk 1]
```diff
@@ -63,6 +64,9 @@ export default function reducer(state :Immutable.Map<*, *> = INITIAL_STATE, acti
     case actionTypes.SUBMIT_TOP_UTILIZERS_FAILURE:
       return state.set('isGettingResults', false);
 
+    case actionTypes.GET_TOP_UTILIZERS_NEIGHBORS_SUCCESS:
+      return state.set('neighbors', action.neighbors);
```
Comment (UnsungHero97) [inline src/containers/toputilizers/TopUtilizersReducer.js]: Use `Immutable.fromJS(action.neighbors)` instead in order to store the data as an immutable object.

File: src/containers/toputilizers/containers/TopUtilizersResultsContainer.js [hunk 2]
```diff
@@ -115,7 +174,8 @@ function mapStateToProps(state) {
     results: topUtilizers.get('topUtilizersResults'),
     isGettingResults: topUtilizers.get('isGettingResults'),
     associations: topUtilizers.get('associations'),
-    topUtilizersDetails: topUtilizers.get('topUtilizersDetailsList').toJS()
+    topUtilizersDetails: topUtilizers.get('topUtilizersDetailsList').toJS(),
```
Comment (UnsungHero97) [inline src/containers/toputilizers/containers/TopUtilizersResultsContainer.js]: Don't do `.toJS()`.

Comment (katherinebernstein) [inline src/containers/toputilizers/containers/TopUtilizersResultsContainer.js]: Can you pass an immutable object in an API call?

**Comment by geekbeast** [comment]
For demo! @UnsungHero97 Katherine is filing to ticket to fix this.

**Pull request merged**